src/v2/Apps/Order/Routes/__tests__/NewPayment.jest.tsx

```diff
@@ -71,24 +87,26 @@ const testOrder: NewPaymentTestQueryRawResponse["order"] = {
 }
 
 describe("Payment", () => {
+  const pushMock = jest.fn()
+  let isCommittingMutation
+
   beforeAll(() => {
     window.sd = { STRIPE_PUBLISHABLE_KEY: "" } as GlobalData
   })
-  const { buildPage, mutations, routes, ...hooks } = createTestEnv({
-    Component: NewPaymentFragmentContainer,
-    defaultData: {
-      order: testOrder,
-      me: { creditCards: { edges: [] } },
-      system: {
-        time: {
-          unix: 222,
-        },
-      },
-    },
-    defaultMutationResults: {
-      ...creatingCreditCardSuccess,
-      ...fixFailedPaymentSuccess,
-    },
+
+  const { getWrapper } = setupTestWrapper({
+    Component: (props: any) => (
+      <MockBoot>
+        {/* @ts-ignore */}
```

Comment:
I don't like so many HoCs wrapping the FragmentContainer and the exposed props for tests get complicated to handle, sorry about introducing this ts-ignore here